frontend/src/context/sidebarStateSlice.ts

```diff
@@ -0,0 +1,30 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import type { RootState } from './store';
+
+type SidebarState = {
```

Comment:
Is `sideBar` the proper name for this? sideBar could be many things.

Reply:
renamed